datarockets-style.gemspec

```diff
@@ -32,10 +32,10 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "rubocop", "~> 1.56.3"
+  spec.add_dependency "rubocop", "~> 1.65.1"
   spec.add_dependency "rubocop-graphql", "~> 1.4"
```

Comment:
How about upgrading this to [v1.5.4](https://github.com/DmitryTsepelev/rubocop-graphql/blob/v1.5.4/CHANGELOG.md#154-2024-08-03)?

Reply:
Actually, the goal of this PR is to fix [the issue](https://github.com/datarockets/ruby-style/issues/279).
I don't see any reason to update dependencies that not related to the bug.

Reply:
@Set27 Yeah, but you're going to make new release, why not to upgrade all dependencies to the latest version then and provide the newest functionality to other users with this release?

Reply:
@an-korn done